pyanaconda/addons.py

```diff
@@ -27,7 +27,7 @@
 from pyanaconda.progress import progress_message
 from pyanaconda.i18n import N_
 
-log = logging.getLogger("anaconda")
+log = logging.getLogger("anaconda.addons")
```

Comment:
Have you experimented with `%(module)s`? 
https://docs.python.org/3/library/logging.html#logrecord-attributes